core/modules/map.rs

```diff
@@ -670,15 +665,18 @@ impl ModuleMap {
   /// Implementors must manually call [`JsRuntime::run_event_loop`] to drive
   /// module evaluation future.
   ///
-  /// `Error` can usually be downcast to `JsError` and should be awaited and
-  /// checked after [`JsRuntime::run_event_loop`] completion.
+  /// Rejection errors from module evaluation are treated as unhandled rejections, and
```

Comment:
"Rejection errors" could be worded better, I don't really understand what it means here.

Reply:
Removed this comment -- JsRuntime has the primary one now.